src/main/scala/wdl4s/WdlSyntaxErrorFormatter.scala

```diff
@@ -45,6 +45,18 @@ case class WdlSyntaxErrorFormatter(terminalMap: Map[Terminal, WdlSource]) extend
      """.stripMargin
   }
 
+  // TODO: these next two methods won't be called by the parser because there are no lists in the WDL grammar that
+  // cause these to be triggered.  Currently the parser is passing in 'null' for the value of 'last' and when that
+  // changes, these errors can be made more helpful.
+
+  def missingListItems(method: String, required: Int, found: Int, last: Terminal): String = {
+    s"ERROR: $method requires $required items, but only found $found"
+  }
+
+  def missingTerminator(method: String, terminal: TerminalIdentifier, last: Terminal): String = {
+    s"ERROR: $method requires a terminator after each element"
```

Comment:
not sure if this is covered by your disclaimers, but the second and third parameters aren't used